src/lib/eina/eina_types.h

```diff
@@ -285,6 +285,8 @@
 # endif
 # define EINA_MALLOC
 # define EINA_PURE
+# define EINA_HOT
+# define EINA_COLD
```

Comment:
What are those for?

Reply:
see b0a102a

Reply:
Those came from my branch. They were needed but not defined for Windows. Under GCC they are defined to use GCC-specific attributes (`__attribute__(hot)` and `__attribute__(cold)`), for optimization purposes.